### ISSUE TITLE
Inconsistent sourcing from Google Maps

### ISSUE THREAD

**Issue by SrihariThalla** [mapbox/mapping#299]
Hi!

I've just observed an ATM being sourced from Google Maps. This was about 5 years ago, though!
Can someone look into the changeset/user and check if there are other objects being sourced from Google Maps?

ATM - https://www.openstreetmap.org/node/1824367470

Thanks!

**Comment by krishnanammala** [comment]
Hi @SrihariThalla ,

Thanks for filing a ticket and letting know the issue. 

I looked into the user edits and all the edits seems good except **2** edits sourcing from Google Maps. 
1. https://www.openstreetmap.org/node/1824367470
2. https://www.openstreetmap.org/way/171388855

He mostly mapped in the city of [Visakhapatnam](https://en.wikipedia.org/wiki/Visakhapatnam) like adding roads, parks, POI's, hospitals, colleges etc. I am also from this city , so it helped me validating his very easily because of my local knowledge. I fixed some of his random edits and regarding the Google source issues , I commented on his [changeset](https://www.openstreetmap.org/changeset/12206698#map=15/17.7328/83.3274) letting know the user about proprietary sources in OpenStreetMap (OSM).
